Making `method` and `methods` optional (#2128)
RobinTail/express-zod-api: README.md

## Multiple schemas for one route

Thanks to the `DependsOnMethod` class a route may have multiple Endpoints attached depending on different methods.
It can also be the same Endpoint that handles multiple methods as well. The `method` and `methods` property can be

> omitted when in `EndpointsFactory::build()` so that the method determination could be delegated to the `Routing`.

Author: ```suggestion
omitted for `EndpointsFactory::build()` so that the method determination would be delegated to the `Routing`.
```

```typescript
import { DependsOnMethod } from "express-zod-api";

const endpointA = factory.build({ methods: ["get", "delete"] });

> const endpointB = factory.build({ methods: ["post", "patch"] });

Author: no longer needed, but should explain how it works

Author: done 6f77578c
